docker-compose.test.yml

```diff
@@ -135,6 +137,7 @@ services:
         KERAS_PACKAGE: keras==2.3.1
         PYTORCH_PACKAGE: torch==1.6.0+cu101
         TORCHVISION_PACKAGE: torchvision==0.7.0+cu101
+        PYTORCH_LIGHTNING: pytorch_lightning==1.2.6
```

Comment:
Are there cuda versions for pytorch_lightning, or is this pure python on top of the respective pytorch package (which has the cuda bit)?

Reply:
This is pure python on top, it will relay on horovod backend and torch for GPU training.